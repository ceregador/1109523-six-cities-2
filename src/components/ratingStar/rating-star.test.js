import React from 'react';
import RatingStar from '../ratingStar/rating-star.jsx';
import Renderer from 'react-test-renderer';

it(`renders correctly`, () => {
  const tree = Renderer
    .create(
        <RatingStar
          title={``}
          rate={1}
          isToggled={false}
          onClick={() => null}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
