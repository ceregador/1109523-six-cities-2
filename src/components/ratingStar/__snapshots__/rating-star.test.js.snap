// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="reviews__rating-label form__rating-label"
  htmlFor="1-star"
  onClick={[Function]}
  title=""
>
  <input
    className="form__rating-input visually-hidden"
    id="1-star"
    name="rating"
    type="radio"
    value={1}
  />
  <svg
    className="form__star-image"
    height="33"
    style={
      Object {
        "fill": "#c7c7c7",
      }
    }
    width="37"
  >
    <use
      xlinkHref="#icon-star"
    />
  </svg>
</label>
`;
